### ISSUE TITLE
Eastman PP region schematic

### ISSUE THREAD

**Issue by alanccai** [vicelab/sierra-pywr#25]
Comparing   this section of the network to the USGS schematic, there are some   inconsistencies.  Specifically, does   Eastwood PP draw its water from Balsam Creek (as suggested by the network) or   from Tunnel No. 7 (as suggested by the USGS schematic)?          Also, does the outflows from Eastwood PP flow back into NF Stevenson Creek   (network) or directly into Shaver Lake (schematic)?          Regardless of these things, the IFR node for bl Tunnel 7 Div looks in the   right place.  The USGS Gauge for this   IFR, 11239300, isn't currently in the network, but could be added.

**Comment by rheinheimer** [comment]
Overall the schematic in OpenAgua appears to be correct. The USGS schematic is fairly course resolution. We can look at satellite imagery for the time being, but I'm sure other schematics will confirm that the USGS schematic is misleading.

Regarding 11239300, that is a little too within the catchment to be meaningful for us.

**Comment by rheinheimer** [comment]
@alanccai I'm not closing this quite yet. Can you follow up and look for a proper schematic of this system for confirmation? Here are the changes I've made. Note Thomas A. Edison is in the lower left. Eastman PP releases into NF Stevenson before TAE.

![image](https://user-images.githubusercontent.com/9124277/68735077-23b31080-05a2-11ea-84a3-e9474c77c0eb.png)

**Comment by alanccai** [comment]
Here is USGS's description of 11238250 Eastwood Powerplant above Shaver Lake, near Big Creek, CA

 - Flow is diverted from Huntington Lake (station 11236000) and Pitman Creek (station 11237600) to Balsam Meadows Forebay, then through a tunnel to the powerplant. The powerplant turbine can be reversed and pump water back up to Balsam Meadows Forebay. Water is returned to Shaver Lake (station 11239500) 0.25 mi downstream for further power development in Big Creek powerplants.

**Comment by alanccai** [comment]
![image](https://user-images.githubusercontent.com/35709110/68738534-6a513c80-059b-11ea-9313-6bb05701b075.png)
This is how it's named in the 2000 SCE document.

It says even more specifically about the location of the IFR: "Intersection of No. Fk. Stevenson Creek and Shaver Lake perimeter road."

**Comment by alanccai** [comment]
So, from Google Maps, it looks like it's right here:

![image](https://user-images.githubusercontent.com/35709110/68738741-f2cfdd00-059b-11ea-84e5-f78686b3efff.png)

**Comment by alanccai** [comment]
I'm sorry I couldn't find a schematic.

**Comment by rheinheimer** [comment]
Here's my latest revision. I added the gauge, but we'll need to recognize that this is pretty far from the IFR, so it isn't an exact match, but better than nothing. For future reference, you can see the gauge location on the USGS website. E.g.: https://waterdata.usgs.gov/nwis/nwismap/?site_no=11239300

I moved the outflow from Eastman PP to lower down; we may change this so it all flows into the Shaver Lake node. However, mathematically it would be the same as is (and there is a routine that automatically simplifies the network anyway), so this is a low priority.

My last remaining question is about the Tunnel 7 inflow to NF Stevenson Cr., the one that crosses Tollhouse Rd. in the figure below. Is this real? I want to assume so, but would also like verification.

![image](https://user-images.githubusercontent.com/9124277/68739815-60d1cf80-05af-11ea-9823-1c809dd77f1b.png)

**Comment by alanccai** [comment]
I actually think the descriptions line up well for 11239300.  SCE describes it as "Intersection of No. Fk. Stevenson Creek and Shaver Lake perimeter road."  USGS describes it as North Fork Stevenson Creek at Perimeter Road, near Big Creek, CA and "100 ft upstream from Perimeter Road,"

**Comment by alanccai** [comment]
![image](https://user-images.githubusercontent.com/35709110/68742137-3d555780-05a4-11ea-84c4-12d48fe971ed.png)

**Comment by alanccai** [comment]
https://books.google.com/books?id=JOw0AQAAMAAJ&newbks=1&newbks_redir=0&dq=%22Tunnel+7%22+%22huntington+lake%22&source=gbs_navlinks_s

**Comment by rheinheimer** [comment]
From the 2000 Big Creek ALP doc, the 11239300 gauge is specifically for the No. Fk. Stevenson Creek above Shaver Lake IFR. So, I moved the IFR downstream, and renamed it. See below.

Importantly, we now need to note that the gauge/IFR are not at either 1) a purely controlled outlet or 2) at a subcatchment outlet. So any results there will need to be taken in that context. Furthermore, it seems the last bit of the No. 7 tunnel is normally closed.

Closing this issue since the schematic is correct.

![image](https://user-images.githubusercontent.com/9124277/68781231-1414e580-05fd-11ea-8426-74b6d0df53c6.png)
